modules/onboarding/README.md

```diff
@@ -0,0 +1,78 @@
+# Oracle Cloud Onboarding Module
+
+This module will deploy foundational onboarding resources in Oracle for a compartment or root tenancy.
+
+The following resources will be created in each instrumented compartment/tenancy:
+
+- An Admit Policy on the target tenant that will allow sysdig tenant to `inspect` compartments in the specified
+  compartment/tenancy.
+- A cloud account in the Sysdig Backend, associated with the specified compartment/tenant and with the required
+  component to serve the foundational functions.
+- A cloud organization in the Sysdig Backend, associated with the specified compartment/tenant to fetch the organization
+  structure(compartment tree) to install Sysdig Secure for Cloud.
+
+Note:
+
+- The outputs from the foundational module, such as `sysdig_secure_account_id` are needed as inputs to the other
+  features/integrations modules for subsequent modular installations.
+
+<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+
+## Requirements
+
+| Name                                                                      | Version   |
+|---------------------------------------------------------------------------|-----------|
+| <a name="requirement_terraform"></a> [terraform](#requirement\_terraform) | >= 1.0.0  |
+| <a name="requirement_oci"></a> [oci](#requirement\_oci)                   | >= 6.19.0 |
+| <a name="requirement_sysdig"></a> [sysdig](#requirement\_sysdig)          | >= 1.34.0 |
```

Comment:
nit: same comment as above for sysdig provider versioning.

Reply:
same here.

Reply:
done